docs/web/howto/client.rst

```diff
@@ -331,74 +331,85 @@ This function returns a ``Deferred`` that fires with the body after the request
 
 
 
-HTTP over SSL
-~~~~~~~~~~~~~
-
-
-    
-
-Everything you've read so far applies whether the scheme of the request
-URI is *HTTP* or *HTTPS* .  However, to control the SSL
-negotiation performed when an *HTTPS* URI is requested, there's
-one extra object to pay attention to: the SSL context factory.
-
-
-
-
-
+Customizing your HTTPS Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``Agent`` 's constructor takes an optional second argument, a context factory.
-Generally you don't want to create the context factory yourself: it's easy to get wrong.
-Instead, Twisted provides helpers for building correctly configured ones.
-For example, you can use :api:`twisted.internet.ssl.optionsForClientTLS <optionsForClientTLS>`, which takes care of all hostname certificate verification for you.
+Everything you've read so far applies whether the scheme of the request URI is *HTTP* or *HTTPS* .
 
+By default, since version 15.0.0 (if you're using an earlier version, please upgrade, as this is a critical security feature!), Twisted will automatically validate ``https`` URLs against your platform's trust store to the best of its ability.
+Be sure to ``pip install twisted[tls]`` in order to get all the dependencies necessary to do TLS properly, but beyond that you should not need to do much.
 
+However, for some uses, you may need to customize Agent's use of HTTPS; for example, to provide a client certificate, or to use a custom certificate authority for an internal network.
 
+Here, we're just going to show you how to inject the relevant TLS configuration into an Agent, so we'll use the simplest possible example, rather than a more useful but complex one.
 
+``Agent``'s constructor takes an optional second argument, which allows you to customize its behavior with respect to HTTPS.
```

Comment:
> wrt grammar citations, I've long thought we should officially adhere to a particular style guide. Then folks would have something to look at to figure this stuff out in advance and there'd be no confusion about where the rules come from (and we really don't want to allow just any random citation as justification since there are conflicting rules).

I definitely agree. The key thing that's stopped me from doing exactly that is that the two style guides I might want to adopt, http://www.apstylebook.com or http://www.chicagomanualofstyle.org/home.html, both require a site-license.

Is there a good freely-available style guide we could adopt?